### PR TITLE
Fix for issue #2800.

### PR DESCRIPTION
**Pull request by djkevincr**

### CODE REVIEW

File: modules/ballerina-core/src/main/java/org/ballerinalang/util/semantics/SemanticAnalyzer.java [hunk 1]
```diff
@@ -2610,7 +2610,12 @@ private void visitLExprsOfAssignment(AssignStmt assignStmt, Expression[] lExprs)
 
                 SimpleVarRefExpr refExpr = (SimpleVarRefExpr) expr;
                 String varName = refExpr.getVarName();
```
Comment (sameera-jayasoma) [inline modules/ballerina-core/src/main/java/org/ballerinalang/util/semantics/SemanticAnalyzer.java]: We should throw an error, if both variables have been already declared.

Comment (djkevincr) [inline modules/ballerina-core/src/main/java/org/ballerinalang/util/semantics/SemanticAnalyzer.java]: +1

File: modules/ballerina-core/src/main/resources/MessagesBundle.properties [hunk 2]
```diff
@@ -124,3 +124,4 @@ incorrect.action.invocation = invalid action invocation. connector variable expe
 invalid.var.assignment = invalid usage of 'var'
 server.connector.already.exist = server connector config with port - {0} already exist with different parameters
 invalid.service.protocol = server connector does not exist for the package ''{0}''
+ignored.var.assignment = var keyword declared assignment statement should have at least one non declared variable
```
Comment (sameera-jayasoma) [inline modules/ballerina-core/src/main/resources/MessagesBundle.properties]: no.new.variables.var.assignment = no new variables on left side

Comment (djkevincr) [inline modules/ballerina-core/src/main/resources/MessagesBundle.properties]: +1

**Comment by djkevincr** [comment]
Addressed all the comments.

**Pull request merged**